packages/web/components/PageHeader.tsx

```diff
@@ -217,9 +243,30 @@ export const PageHeader: React.FC = () => {
             right="0"
             onClick={onClose}
           />
-          <Box position="absolute" left="calc(50% - 16.5rem)" top="0">
-            <NextImage src={MetaBox} alt="MetaBox" width={528} height={695} />
-          </Box>
+
+          <Box
+            position="absolute"
+            left="calc(50% - 16.5rem)"
+            top="0"
+            width={522}
+            height={680}
+            background={
+              isBackdropFilterSupported()
+                ? `linear-gradient(180deg, rgba(76, 63, 143, 0.75) 62.76%, rgba(184, 169, 255, 0.75) 100%)`
+                : `linear-gradient(180deg, rgba(76, 63, 143, 1.00) 62.76%, rgba(184, 169, 255, 1.00) 100%)`
+            }
+            style={{ backdropFilter: 'blur(10px)' }}
+          />
+          <Image
+            position="absolute"
+            left="calc(50% - 16.5rem)"
+            top={678}
+            src={MetaGradientBottom}
+            alt="MetaGradientBottom"
+            width={522}
```

Comment:
Shouldn't use hardcoded values. Also, Chakra has a native Gradient component now, shouldn't use an image since it takes a lot more bandwidth to load vs plain CSS.

Reply:
The image is the bottom cap on the drawer. The background is a CSS gradient.